backend-api/src/main/java/pms/api/HelloWorldController.java

```diff
@@ -0,0 +1,20 @@
+package pms.api;
+
+import org.springframework.http.MediaType;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RestController;
+
+import java.util.List;
+
+@RestController
+@RequestMapping(value = "/v1/hello", produces = MediaType.APPLICATION_JSON_UTF8_VALUE)
+public class HelloWorldController {
+    @GetMapping()
+    public ResponseEntity<String> get() {
+        return ResponseEntity.ok()
+                .header("Custom-header", "foo")
+                .body("World");
+    }
```

Comment:
I'm newish to spring boot, will `@RequestMapping(value = "/v1/hello", produces = MediaType.APPLICATION_JSON_UTF8_VALUE)` make it so that this endpoint returns an `application/json` content type header in the response? 

Will `.body("World")` make spring boot return a just a plain string `World` as the response? Wouldn't this be incompatible with the json content type header defined previously?

Reply:
Fixed this, created a new class called message. Now it does not show the content type error.